docs/user/metal/install_upi.md

```diff
@@ -320,4 +320,4 @@ terraform destroy -auto-approve
 [upi-metal-example-tfvar]: ../../../upi/metal/terraform.tfvars.example
 [upi-metal-example]: ../../../upi/metal/README.md
 [openshift-upi-metal]: https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-bare-metal.html
-[upi-metal-prereqs]: https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-bare-metal.html#prerequisites
\ No newline at end of file
+[upi-metal-prereqs]: https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-bare-metal.html#prerequisites
```

Comment:
nit: I think this can be dropped. I pulled this out and did a diff and didn't find anything different